ts-sdk/whirlpool/tests/token.test.ts

```diff
@@ -207,32 +146,23 @@ describe("Token Account Creation", () => {
     setSolWrappingStrategy("none");
 
     const result = await prepareTokenAccountsInstructions(rpc, signer, {
-      [TOKEN_MINT_1]: 100n,
-      [TOKEN_MINT_2]: 100n,
+      [mintA]: 100n,
+      [mintB]: 100n,
       [NATIVE_MINT]: 100n,
```

Comment:
Just to be sure, in this case (strategy is none), the balance of ATA for native mint should be zero even if 100n is specified.

Reply:
Yep. This is how it is currently explained in the docs:
```
 * - **None**:
 *   Uses or creates the ATA without performing any SOL wrapping or unwrapping.
 ```

It might make sense to throw if the balance is insufficient (also for the non-SOL mints). Basically this can be used to say: "I need x amount in this token account" and is currently only doing something for SOL (if wrapping strategy is not none). What do you think?

Reply:
Thanks for the explanation!
It makes sense🙂 (memo: the instruction requires X SOL, but my ATA already has >  X SOL, so no need to wrap)